src/utils/helpers.js

```diff
@@ -0,0 +1,15 @@
+export function setButtonText(
+  btn,
+  isLoading,
+  defaultText = "Save",
+  loadingText = "Saving..."
+) {
+  if (isLoading) {
+    console.log(`Setting text to ${loadingText}`);
+    btn.textContent = "Saving...";
+    //set the loading text
+  } else {
+    //set not loading text
+    btn.textContent = "Submit";
```

Comment:
Please, use `defaultText` and `loadingText` to be able the reassign the texts if needed


```js
export function setButtonText(
  btn,
  isLoading,
  defaultText = "Save",
  loadingText = "Saving..."
) {
  if (isLoading) {
    btn.textContent = loadingText;
  } else {
    btn.textContent = defaultText;
  }
}

```